test/canvas/canvasSpec.js

```diff
@@ -125,10 +125,10 @@ describe('Canvas', function () {
     expect(canvas.shapes[1].id).to.equal('shape-1');
     expect(canvas.shapes[1].shape).to.eql(obstacleTop);
     expect(canvas.shapes[1].type).to.equal('Obstacle');
-  
+
   });
 
-  it('#moveObstacles', function () {
+  it('#collisionDetection', function () {
```

Comment:
Shouldn't this test call "moveObstacles" and the below one "collisionDetection"?